server/views/pages/appointments/attendance/attendees.njk

```diff
@@ -0,0 +1,183 @@
+                <span class="govuk-caption-l govuk-!-margin-bottom-4">{{ showLocation(appointment) }}</span>
+            {% else %}
+                <h1 class="govuk-heading-l govuk-!-margin-0" data-qa='heading'>Record attendance at {{ appointments | length }} appointments</h1>
+                <span class="govuk-caption-l" data-qa="date-caption">{{ appointments[0].startDate | toDate | formatDate }}</span>
+            {% endif %}
+
+            <h2 class="govuk-heading-m">{{ attendanceSummary.attendeeCount }} attendee{{ "s" if attendanceSummary.attendeeCount != 1 }}</h2>
+
+            {% if attendanceSummary.attendeeCount > 1 %}
+                <ul class="govuk-list">
+                    <li>
+                        <span>Attended:</span>
+                        <span class="govuk-!-font-weight-bold">{{ attendanceSummary.attended }}</span> ({{ attendanceSummary.attendedPercentage }}%)
+                    </li>
+                    <li>
+                        <span>Did not attend:</span>
+                        <span class="govuk-!-font-weight-bold">{{ attendanceSummary.notAttended }}</span> ({{ attendanceSummary.notAttendedPercentage }}%)
+                    </li>
+                    <li>
+                        <span>Not recorded yet:</span>
+                        <span class="govuk-!-font-weight-bold">{{ attendanceSummary.notRecorded }}</span> ({{ attendanceSummary.notRecordedPercentage }}%)
+                    </li>
+                </ul>
+            {% endif %}
+        </div>
+    </div>
+
+    {% set attendeeListRows = [] %}
+    {% for appointment in appointments %}
+        {% for attendee in appointment.attendees %}
+            {% set prisoner = attendee.prisoner %}
+            {% set items = [
+                    {
+                        html: showProfileLink({
+                            firstName: prisoner.firstName,
+                            middleNames: prisoner.middleNames,
+                            lastName: prisoner.lastName,
+                            prisonerNumber: prisoner.prisonerNumber,
+                            inCaseLoad: prisoner.prisonCode == user.activeCaseLoadId
+                        }),
+                        attributes: {
+                            'data-qa': 'prisoner-' + loop.index + '-name-and-number',
+                            "data-sort-value": prisoner | formatName(NameFormatStyle.lastCommaFirstMiddle, false)
+                        }
+                    },
+                    {
+                        text: statusBasedCellLocation(prisoner.cellLocation, prisoner.status, prisoner.prisonCode == user.activeCaseLoadId),
+                        attributes: { 'data-qa': 'prisoner-' + loop.index + '-location' }
+                    }
+                ]
+            %}
+            {% if not isSingleAppoinment %}
+                {% set items = items.concat(
+                    {
```

Comment:
Don't show appointment column if it is only obe appointment as details will be shown higher up.